phoenix-core/src/main/java/org/apache/phoenix/monitoring/MetricType.java

```diff
@@ -208,23 +209,23 @@ private MetricType(String shortName, String description, LogLevel logLevel, PDat
     public String description() {
         return description;
     }
-    
```

Comment:
nitpick : there are various changes fixing the extra spaces not part of the change - should we skip this as part of this change ?

Reply:
Agree, good to keep only addition of `MUTATION_SYSCAT_TIME` for this patch